sample/HttpTrigger-CSharp-CustomRoute/run.csx

```diff
@@ -8,10 +8,10 @@ using Microsoft.Azure.WebJobs.Host;
 public class ProductInfo
 {
     public string Category { get; set; }
-    public int Id { get; set; }
+    public int? Id { get; set; }
 }
 
-public static ProductInfo Run(ProductInfo info, string category, int id, TraceWriter log)
+public static ProductInfo Run(ProductInfo info, string category, int? id, TraceWriter log)
```

Comment:
Had to make core SDK changes to support nullable bindings like this. See https://github.com/Azure/azure-webjobs-sdk/pull/869